app/liquid_tags/user_subscription_tag.rb

```diff
@@ -214,22 +219,22 @@ class UserSubscriptionTag < LiquidTagBase
     }
   JAVASCRIPT
 
-  def initialize(_tag_name, cta_text, _tokens)
+  def initialize(_tag_name, cta_text, parsed_context)
```

Comment:
Either the [`liquid` gem](https://github.com/Shopify/liquid) has made some updates or we just made a slight mistake in development. You might notice that in every other liquid tag, the third argument for `initialize` is `tokens`. I dug through the source code and found that's incorrect - [it's actually `pasred_context`](https://github.com/Shopify/liquid/blob/1feaa6381300d56e2c71b49ad8fee0d4b625147b/lib/liquid/tag.rb#L27).

I will update all the other liquid tags in a separate PR since that will result in like 30+ file changes.

Reply:
After looking at the source code you linked to, it looks like it actually might be `parse_context` rather than `parsed_context` - am I seeing that correctly? 😬

Reply:
Good catch, thanks! I'll update it.